static-assets/components/cstudio-contextual-nav/wcm_dropdown.js

```diff
@@ -49,7 +49,8 @@ CStudioAuthoring.ContextualNav.WcmDropDown = CStudioAuthoring.ContextualNav.WcmD
     var mainContainerEl = YDom.get('acn-dropdown-wrapper');
 
     var navBarSiteNameEl = YDom.get('navbar-site-name');
-    navBarSiteNameEl.innerHTML = CStudioAuthoringContext.site;
+    const siteName = CrafterCMSNext.system.store.getState().sites.byId[CStudioAuthoringContext.site].name;
+    navBarSiteNameEl.innerHTML = siteName??CStudioAuthoringContext.site;
```

Comment:
Don't use these operators (??, ?) on legacy code. We need to give it a little more time — almost there, though. 🙂